zcash_client_sqlite/src/wallet/init.rs

```diff
@@ -574,35 +574,43 @@ mod tests {
                 fee INTEGER,
                 FOREIGN KEY (block) REFERENCES blocks(height)
             )",
-            "CREATE TABLE transparent_received_output_spends (
+            r#"CREATE TABLE "transparent_received_output_spends" (
                 transparent_received_output_id INTEGER NOT NULL,
                 transaction_id INTEGER NOT NULL,
                 FOREIGN KEY (transparent_received_output_id)
-                    REFERENCES utxos(id)
+                    REFERENCES transparent_received_outputs(id)
                     ON DELETE CASCADE,
                 FOREIGN KEY (transaction_id)
                     -- We do not delete transactions, so this does not cascade
                     REFERENCES transactions(id_tx),
                 UNIQUE (transparent_received_output_id, transaction_id)
+            )"#,
+            "CREATE TABLE transparent_received_outputs (
+                id INTEGER PRIMARY KEY,
+                transaction_id INTEGER,
```

Comment:
I have actively been attempting to move us to the normal SQL naming conventions:
* tables are named `<plural_noun>`
* the primary key is always named `id`
* foreign key columns are always named `<singular_noun>_id`

Reply:
Oh this is analogous to `{sapling,orchard}_received_note_spends(transaction_id)`. Never mind, this is fine as-is.

Reply:
I explicitly did not use the normal SQL naming convention when originally writing the tables, because (for instance) `transaction_id` collides semantically with "transaction ID". But Kris spent enough time convincing me that I'm now not opposed to this migration.